src/js/vic-v2/components/CropperController.jsx

```diff
@@ -416,15 +438,15 @@ export default class CropperControls extends React.Component {
           cropstart={this.onCropstart}
           cropend={this.onCropend}
           cropmove={() => true}
-          minContainerHeight={this.props.narrowLayout ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE}
+          minContainerHeight={this.state.smallCropBox ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE}
```

Comment:
Since this is an aspect of the canvas, it has to be managed with js instead of css.